config/cfroutesync/commons.lib.yaml

```diff
@@ -0,0 +1,10 @@
+#@ load("@ytt:data", "data")
+
+#@ def labels(instance="abcd"):
+app: cfroutesync
+app.kubernetes.io/name: cfroutesync
+app.kubernetes.io/instance: #@ instance
```

Comment:
not sure what does this label should contain

Reply:
Hey @mike1808 :octocat: 

 I am thinking about closing this PR and creating a story to explore what labels we should add to cfroutesync that follow cf-for-k8s/kubernetes guidelines. What do you think? 

Related Slack thread: https://pivotal.slack.com/archives/CM4PTJ8C9/p1585693746070100?thread_ts=1585692794.069200&cid=CM4PTJ8C9